inference-engine/thirdparty/clDNN/src/include/reorder_inst.h

```diff
@@ -46,7 +46,7 @@ struct typed_program_node<reorder> : public typed_program_node_base<reorder> {
 
 private:
     bool req_reinterpr = false;
-    tensor input_offset = tensor{0};  // used by reorder to winograd domain
+    tensor pad = tensor{0};  // used by reorder to winograd domain
```

Comment:
As I can see, this pad is always equal to 0. Do we really need it?

Reply:
Removed